ldi-core/geojson-to-wkt/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/converter/GeoJsonConverter.java

```diff
@@ -0,0 +1,18 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldi.converter;
+
+import org.apache.jena.datatypes.RDFDatatype;
+import org.apache.jena.datatypes.TypeMapper;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.Statement;
+
+import java.util.List;
+
+public interface GeoJsonConverter {
+    String GEOSPARQL_URI = "http://www.opengis.net/ont/geosparql";
+
+    List<Statement> createNewGeometryStatements(Statement statement, Model geometryModel);
+
+    default RDFDatatype getWktLiteralDataType() {
```

Comment:
Can this be a static method rather than e default one? defaults should only be used when dealing with legacy code